crates/anstyle-svg/src/lib.rs

```diff
@@ -305,7 +305,7 @@ impl Term {
         use std::fmt::Write as _;
 
         let br = if self.use_html5 { "<br>" } else { "<br />" };
-        writeln!(buffer, r#"  <div class="container {FG}">"#).unwrap();
+        write!(buffer, r#"<div class="container {FG}">"#).unwrap();
```

Comment:
Please revert this commit

I doubt this makes enough of a difference to be worth it and part of the intent for `ansstyle-svg` is to be usable for tests which involves diffing the content